dev-java/jna/jna-5.13.0.ebuild

```diff
@@ -102,6 +102,7 @@ src_compile() {
 	cd native || die
 	local args=(
 		CC="$(tc-getCC)"
+		STRIP=true
```

Comment:
does this really prevent stripping the library?

Reply:
> does this really prevent stripping the library?

Need to find out how to reproduce that QA warning, then I can answer. sorry.